### DISCUSSION TITLE
Support for 20 frames per second

### DISCUSSION THREAD

**Discussion by MenahemOwlytics** [AxisCommunications/acap-computer-vision-sdk-examples#172] [Q&A]
I use AXIS Q3538-LVE Dome Camera
To capture and process images, I use docker-acap:
Using this code, I try to capture images at 4K resolution and 20 frames per second:

```
VideoCapture cap(1);
cap.set(CAP_PROP_FPS, 20);
cap.set(CAP_PROP_FRAME_WIDTH, IMG_WIDTH);
cap.set(CAP_PROP_FRAME_HEIGHT, IMG_HEIGHT);
cap.set(CAP_PROP_FOURCC, VideoWriter::fourcc('R', 'G', 'B', '3'));
constexpr size_t BUFFERS = 2;

while(true) {
       int index;
       milliseconds startInferenceTime = getCurrentTime();
       cap.read(originalMat);
       milliseconds endInferenceTime = getCurrentTime();
       std::cout << "cap image time: " << endInferenceTime.count() - startInferenceTime.count() << std::endl;
       originalMat.release();
}
```

I'm getting a cap.read() image time of 80ms, so it can only support 12 fps. Can I support 20 fps in this implementation? 

Thanks so much
Menahem

**Comment by Corallo** [comment]
Hi @MenahemOwlytics 

You could try to disable the WDR from the camera settings, that should reduce the capture latency.
